pkg/clusterlink/controllers/node/node_controller.go

```diff
@@ -43,15 +43,15 @@ type Reconciler struct {
 var predicatesFunc = predicate.Funcs{
 	CreateFunc: func(createEvent event.CreateEvent) bool {
 		node := createEvent.Object.(*corev1.Node)
-		return !utils.IsKosmosNode(node)
+		return !utils.IsKosmosNode(node) && !utils.IsExcludeNode(node)
```

Comment:
This solves the problem of the before kosmos install, but in subsequent runs, if a node is changed from a normal node to a exclude node, this can lead to problems or potential bugs. For example, updates related to the node associated with the clustenode will not be synchronized.

Reply:
Yes, currently, this label needs to be added before the controller-manager runs. I have created an issue for it and will further optimize it. #294